helmcharts/README.md

```diff
@@ -0,0 +1,33 @@
+# Eclipse Che Helm Charts
+
+- [Charts](#charts)
```

Comment:
To me none of the headlines was clear. Maybe we should have:
Title
\- Prerequisites (K8s & Helm versions)
\- Installation